### PR TITLE
Differentiate README by person-type

### PR DESCRIPTION
**Pull request by mlangbehn**
README.md now has documentation split up by person-type.

Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

**Comment by eggshell** [comment]
I think we need to have a `getting started` section for both types of people, whereas `contributing` can just stay in the developer section.

**Comment by mlangbehn** [comment]
@eggshell, yeah, that's why I said "more is needed on this one." I'll add [WIP] to the title.

File: README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-- [Contributing](#contributing)
+- [End Users](#end-users)
+- [Developers](#developers)
 
 ## Introduction
 BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
 
-## Getting Started
+## End Users
 Checkout the [getting started](getting_started) section.
```
Comment (eggshell) [inline README.md]: This should be check out.

File: README.md [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
+- [End Users](#end-users)
+- [Developers](#developers)
 
 ## Introduction
 BonnyCI is a Continuous Integration service offered to developers of Open Source projects.
 
-## Getting Started
-Checkout the [getting started](getting_started) section.
+## End Users
+Check out the [end user](end_users) documentation.
```
Comment (eggshell) [inline README.md]: can we make the label and the dir name match up?

File: README.md [hunk 3]
```diff
@@ -3,17 +3,17 @@
 
 ## Table of Contents
 - [Introduction](#introduction)
-- [Getting Started](#getting-started)
-- [Contributing](#contributing)
+- [End Users](#end-users)
```
Comment (eggshell) [inline README.md:6]: Same as below

File: README.md [hunk 4]
```diff
@@ -3,17 +3,17 @@
...
 
-## Getting Started
-Checkout the [getting started](getting_started) section.
+## End Users
+Check out the [end user](end_users) documentation.
 
-## Contributing
-Please see the [contribution documentation](contributing) for more information.
+## Developers
+Please see the [developer documentation](developers) for more information.
```
Comment (eggshell) [inline README.md:16]: Same as above. Also, let's keep the labels consistent. For instance, the word documentation is included in the label here but is not in the end user section.

**Review by eggshell — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (mlangbehn) [reply README.md:6]: This is a link to the section in this README, not to the end_users section of lore, so no, I do not think this should be made to match the directory name.

File: README.md (hunk 2 above)
Comment (mlangbehn) [reply README.md]: We can, but I was making the link text more user friendly.

File: README.md (hunk 4 above)
Comment (mlangbehn) [reply README.md:16]: Same comment as above (making the link text "developers" would make this sentence read poorly). Additionally, including "documentation" in the link text was what I meant to do in both the developer and end user sections.

File: README.md (hunk 2 above)
Comment (eggshell) [reply README.md]: Fair enough, I hadn't read the sentence for grammatical correctness.

File: README.md (hunk 3 above)
Comment (eggshell) [reply README.md:6]: Fair enough, I hadn't read the sentence for grammatical correctness.

Comment (mlangbehn) [reply README.md:6]: Adding summation of chat @eggshell and I just had:
I believe readability of documentation is paramount. @eggshell agreed.

**Review by eggshell — APPROVED** [review]

**Review by eggshell — APPROVED** [review]

**Comment by eggshell** [comment]
lgtm

**Pull request merged**